.github/workflows/ci.yaml

```
@@ -70,7 +70,7 @@ jobs:
       - name: Install docs dependencies
         run: poetry install --with docs
       
-      - name: Check markdown formatting
+      - name: Check markdown, css, toml formatting
```

You still haven't add the CSS formatter
```suggestion
      - name: Check markdown, toml formatting
```

---

This PR only concerns `TOML`, the [Malva PR](https://github.com/bakdata/kpops/pull/385) adds the `css` extension